### ISSUE TITLE
Module 'knossos.ui.hell' not found

### ISSUE THREAD

**Issue by thesourcehim** [ngld/old-knossos#241]
I installed knossos to user directory using:
python3 setup.py build
python3 setup.py install --user --prefix=

When starting I get this error:

Traceback (most recent call last):
  File "/home/thesource/.local/bin/knossos", line 33, in <module>
    sys.exit(load_entry_point('knossos==0.14.2.dev0+1f60d92', 'gui_scripts', 'knossos')())
  File "/home/thesource/.local/bin/knossos", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib64/python3.10/importlib/metadata/__init__.py", line 162, in load
    module = import_module(match.group('module'))
  File "/usr/lib64/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/thesource/.local/lib/python3.10/site-packages/knossos-0.14.2.dev0+1f60d92-py3.10.egg/knossos/launcher.py", line 71, in <module>
    from . import util, ipc, auto_fetch
  File "/home/thesource/.local/lib/python3.10/site-packages/knossos-0.14.2.dev0+1f60d92-py3.10.egg/knossos/ipc.py", line 29, in <module>
    from .windows import ModInstallWindow
  File "/home/thesource/.local/lib/python3.10/site-packages/knossos-0.14.2.dev0+1f60d92-py3.10.egg/knossos/windows.py", line 29, in <module>
    from .ui.hell import Ui_MainWindow as Ui_Hell
ModuleNotFoundError: No module named 'knossos.ui.hell'

**Comment by jg18** [comment]
Hi, sorry, missed the notification for this.

Which distro, distro version, and Python version are you using?

In case you happen to be using Ubuntu or Arch Linux, the instructions are easy: https://fsnebula.org/knossos/

Otherwise, there are some semi-outdated generic build instructions that should point you in the right direction: https://github.com/ngld/old-knossos/wiki/Building-instructions

The latter approach will likely require some trial and error. It's been a long time since I've set up Knossos on *nix.

The TL;DR is that once you have all of the dependencies, including the required Python packages, you can run
```
python3 configure.py
```
and then
```
ninja run
```

CC @ngld
